pkg/usagestats/report.go

```diff
@@ -84,6 +84,7 @@ func buildMetrics() map[string]interface{} {
 		"num_cpu":       runtime.NumCPU(),
```

Comment:
What do you think of removing this in lieu of the new stat?

Reply:
```suggestion
```